setup.py

```diff
@@ -0,0 +1,76 @@
+import setuptools.command.build_py
+import distutils.cmd
+import distutils.log
+import setuptools
+import subprocess
+import os
+
+
+class PyccelCommand(distutils.cmd.Command):
+  """A custom command to run Pyccel on all kernels source files."""
+
+  description = 'run Pyccel on kernels source files'
+  user_options = [
+      # The format is (long option, short option, description).
+      ('language=', None, 'language to pyccelise the kernels files'),
+  ]
+
+  def initialize_options(self):
+    """Set default values for options."""
+    # Each user option must be listed here with their default value.
+    self.language = ''
+
+  def finalize_options(self):
+    """Post-process options."""
+    if self.language:
+      assert self.language in ['fortran', 'c'], (
+          'Language is of the good format' % self.language)
+
+  def run(self):
+    """Run command."""
+
+    psydac_path = os.getcwd()
+
+    if self.language:
+        language_param = '--language ' +  self.language
+    else:
+        language_param = '--language fortran'
+
+    command1 = 'pyccel ' + psydac_path + '/psydac/linalg/stencil2coo_kernels.py '         + language_param
+    command2 = 'pyccel ' + psydac_path + '/psydac/api/ast/transpose_kernels.py ' + language_param
+    command3 = 'pyccel ' + psydac_path + '/psydac/core/field_evaluation_kernels.py '           + language_param
+    command4 = 'pyccel ' + psydac_path + '/psydac/core/bsplines_kernels.py '   + language_param
+
+    self.announce(
+        'Running commands: %s' % str(command1),
+        level=distutils.log.INFO)
+    subprocess.run(command1, check=True, shell=True)
+    self.announce(
+        'Running commands: %s' % str(command2),
+        level=distutils.log.INFO)
+    subprocess.run(command2, check=True, shell=True)
+    self.announce(
+        'Running commands: %s' % str(command3),
+        level=distutils.log.INFO)
+    subprocess.run(command3, check=True, shell=True)
+    self.announce(
+        'Running commands: %s' % str(command4),
+        level=distutils.log.INFO)
+    subprocess.run(command4, check=True, shell=True)
```

Comment:
@yguclu This part of the code don't pass the codacy test. It's complaining about the parameters `shell=True` that could be really problematic. The problem is that to use pyccel, it seems that we need this variable. Do you have any idea on how to solve this ?